test/ctia/bulk/routes_test.clj

```diff
@@ -144,6 +144,14 @@
    :sensor "endpoint.sensor"
    :confidence "High"})
 
+(defn mk-new-identity-assertion [n]
+{:id (str "transient:identity-aasertion-" n)
+ :timestamp #inst "2016-02-11T00:40:48.212-00:00"
+ :valid_time {:start_time #inst "2016-02-01T00:00:00.000-00:00"}
+ :identity {:type "endpoint" :os "Windows 95" :observables [{:type "ip" :value "100.213.110.122"}] :observed_time {:start_time #inst "2016-02-01T00:00:00.000-00:00"}}
+ :assertions [{:name "tag" :value "foo"}]
+ :source "source"})
```

Comment:
That function is not used.
However I would not remove it but should it with test that check bulk operations do work with this new entity type.